pkgs/os-specific/bsd/freebsd/pkgs/iconv.nix

```diff
@@ -0,0 +1,12 @@
+{
+  mkDerivation,
+  libcapsicum,
+  libcasper,
+}:
+mkDerivation {
+  path = "usr.bin/iconv";
```

Comment:
libiconv in all-packages.nix should be updated to use this.  Is that coming later?

Reply:
https://github.com/NixOS/nixpkgs/pull/311789/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22R22488